com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -374,17 +376,28 @@ private static void buildResourceGraph(Project project, IResource resource, Reso
                 throw new CompileExceptionError(resource, 0, "Unable to find resource " + resource.getPath());
             }
             builder.mergeFrom(content);
-            Object message = builder.build();
-            buildResourceGraph(project, (Message) message, currentNode, visitedNodes);
+            Message message = (Message)builder.build();
+            buildResourceGraph(project, message, currentNode, visitedNodes, excludedResources);
+            if (isCollelctionProxy && excludedResources != null) {
+                CollectionProxyDesc desc = (CollectionProxyDesc)message;
+                if (desc.getExclude()) {
```

Comment:
... so I just check if it's excluded collection proxy or not and exclude it if needed